doc/Getting-started/index.md

```diff
@@ -211,8 +211,8 @@ The same address will be configured into Kaa Endpoint SDK libraries generated by
 
 <br>
 
-Navigate to the `Management` page by clicking a link in the upper right corner of the Sandbox web UI.
-Input the desired address in the `Kaa host/IP` section and click the `Update` button.
+Navigate to the **Management** page by clicking a link in the upper right corner of the Sandbox web UI.
```

Comment:
I think something to "Documentation contribution and style guide" for referencing pages and steps.